### PR TITLE
adding details about using the recovery code to login

### PR DESCRIPTION
**Pull request by angelaLazar**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/multifactor-authentication/guardian/user-guide.md [hunk 1]
```diff
@@ -62,19 +62,27 @@ Depending on the type of two-factor authentication you chose when you were signi
 
 After entering your username and password, a push notification will be sent to the Guardian app on your mobile device. This notification is a login request, it includes the application name, the OS and browser of the request, the location and the date of the request. If you recognize this request as your own, tap the **Allow** button. You should now be logged in.
 
+If you do not have your mobile device available to you during sign in, you can enter the recovery code that was given when you signed up by clicking the "enter the code" link.
```
Comment (nikolaseu) [inline articles/multifactor-authentication/guardian/user-guide.md]: The "enter the code" link is for entering the OTP code, not the recovery code. And you need your device, but this helps if you don't have internet or there's a problem with the push notification service.
This is mixing otp code vs recovery code

File: articles/multifactor-authentication/guardian/user-guide.md [hunk 2]
```diff
@@ -62,19 +62,27 @@ Depending on the type of two-factor authentication you chose when you were signi
...
 After entering you enter your email and password, a SMS message will be sent to the phone number you entered when you signed up. Enter the six digit code from the message into the box to complete your login.
 
+If you do not have your mobile device available to you during sign in, you can enter the recovery code that was given when you signed up by clicking the "Use Recovery Code" link.
+
 ## Troubleshooting
 
-### If you do not have your mobile device
+### If you do not have your mobile device or it has no service or its turned off
+
+If you have lost your device or it is unable to recieve notifications, you can finish the two-step authentication with the recovery code from when you signed up. After entering your email and password to login, click the link that says "enter the code" to access your account without using your device.
```
Comment (nikolaseu) [inline articles/multifactor-authentication/guardian/user-guide.md]: This is wrong (the part about the push notification). It's better to leave as it was before.
See comment above.

File: articles/multifactor-authentication/guardian/user-guide.md (hunk 1 above)
Comment (dafortune) [inline articles/multifactor-authentication/guardian/user-guide.md]: Right, also, might worth adding the instruction on how to get the OTP code on your mobile phone.

File: articles/multifactor-authentication/guardian/user-guide.md (hunk 2 above)
Comment (dafortune) [inline articles/multifactor-authentication/guardian/user-guide.md]: Agree

**Comment by dafortune** [comment]
@angelaLazar Let's do I quick call if you want, maybe we were not clear enough about the difference between manually entering otp code vs using a recovery code :)

File: articles/multifactor-authentication/guardian/user-guide.md [hunk 3]
```diff
@@ -62,26 +62,48 @@ Depending on the type of two-factor authentication you chose when you were signi
...
 
+If you do not have your mobile device available to you during sign in, you can enter the recovery code that was given when you signed up by clicking the "Use Recovery Code" link.
+
 ## Troubleshooting
 
-### If you do not have your mobile device
+### If you do not have your mobile device or it is turned off
 
-If you have lost your phone and are unable to finish the two-step authentication you will need the recovery code from when you signed up. Enter this code to access your account without your device.
+If you have lost your device or it is unable to recieve notifications, you can finish the two-step authentication with the recovery code from when you signed up. After entering your email and password to login, click the link that says "Use the recovery code" to access your account without using your device.
```
Comment (nikolaseu) [inline articles/multifactor-authentication/guardian/user-guide.md]: @angelaLazar remove `or it is unable to receive notifications`

File: articles/multifactor-authentication/guardian/user-guide.md [hunk 4]
```diff
@@ -62,26 +62,48 @@ Depending on the type of two-factor authentication you chose when you were signi
 
 After entering your username and password, a push notification will be sent to the Guardian app on your mobile device. This notification is a login request, it includes the application name, the OS and browser of the request, the location and the date of the request. If you recognize this request as your own, tap the **Allow** button. You should now be logged in.
 
+If you are not currently connected to the internet on your mobile device to recieve push notifications, you can enter the OTP(One-time password) manually by clicking "enter the code".
```
Comment (nikolaseu) [inline articles/multifactor-authentication/guardian/user-guide.md]: typo "recieve"

Comment (angelaLazar) [inline articles/multifactor-authentication/guardian/user-guide.md]: Ah thanks, fixed

**Comment by dafortune** [comment]
👍

**Pull request merged**